app/grants/models.py

```diff
@@ -272,6 +272,49 @@ def org_profile(self):
             return profiles.first()
         return None
 
+    @property
+    def history_by_month(self):
+        # gets the history of contributions to this grant month over month so they can be shown o grant details
+        # returns [["", "Subscription Billing",  "New Subscriptions", "One-Time Contributions", "CLR Matching Funds"], ["December 2017", 5534, 2011, 0, 0], ["January 2018", 10396, 0 , 0, 0 ], ... for each monnth in which this grant has contribution history];
```

Comment:
remove comments?

Reply:
hmm... are you sure? im trying to get in the habit of documenting my code more..

Reply:
comments are good! I know we have a lot of docstring style comments which I think are overkill and lead to boilerplate copy-and-pasting but I think comments are something we should definitely encourage, especially if we want outside contributors to grok what we are doing more easily and hence be able to effectively dogfood more @thelostone-mc @owocki

Reply:
alright alright! majority wins :P
the comments stay